@xen-orchestra/web/src/components/vm/dashboard/VmDashboardQuickInfo.vue

```diff
@@ -72,19 +92,21 @@ const { vm } = defineProps<{
 const { t, locale } = useI18n()
 
 const { areVmsReady, getVmHost } = useXoVmCollection()
+const { getPoolById } = useXoPoolCollection()
+const { user } = useXoUserResource({}, () => vm.creation?.user)
 
 const host = computed(() => getVmHost(vm))
 
-// TODO add this to icon when new component is available
-// const hostPowerState = computed(() => {
-//   if (host.value === undefined) {
-//     return
-//   }
-//
-//   return host.value.power_state === 'Running' ? 'running' : 'halted'
-// })
+const pool = computed(() => getPoolById(vm.$pool))
+
+const hostPowerState = computed(() => {
+  if (host.value === undefined) {
+    return 'unknown'
+  }
+
+  return host.value.power_state === 'Running' ? 'running' : 'halted'
```

Comment:
What do you think of using the enum instead?

```suggestion
  return host.value.power_state === HOST_POWER_STATE.Running ? 'running' : 'halted'
```